curvefs/src/client/fuse_client.cpp

```diff
@@ -84,6 +84,16 @@ using rpcclient::MetaCache;
 using common::FLAGS_enableCto;
 
 CURVEFS_ERROR FuseClient::Init(const FuseClientOption &option) {
+    // 读取白名单文件
+    whitelist_config_.SetConfigPath(std::string(
+        getenv("HOME")) + "/curvefs/conf/recycle_whitelist.conf");
+    if (!whitelist_config_.LoadConfig()) {
+        // handle error here
+        LOG(ERROR) << "Failed to open whitelist file: "
+                   << whitelist_config_.GetConfigPath();
+        return CURVEFS_ERROR::INTERNAL;
```

Comment:
I think if we can not load the whitelist config file, we do not need to exit the fuse client, just log a warning message is OK, because the whitelist is not _MUST_ configed.